idaes/models/unit_models/pressure_changer.py

```diff
@@ -444,7 +459,7 @@ def add_adiabatic(self):
 
         @self.Constraint(self.flowsheet().time)
         def zero_work_equation(b, t):
-            return self.control_volume.work[t] == 0
+            return b.control_volume.work[t] == 0
```

Comment:
I would argue this is the reverse of what you should do - it would probably be better to have `self` as the argument in both cases.

Also, whilst not part of this PR, this option is misnamed. `adiabatic` means no heat transfer (which is true for all our options), not no work transfer.

Reply:
I agree. All I did was update `self` to `b` on a whim and has no relevance to the PR, so I can just change it back.